crates/db_views/src/post_view.rs

```diff
@@ -387,13 +387,16 @@ fn queries<'a>() -> Queries<
 
     if let Some(search_term) = &options.search_term {
       let searcher = fuzzy_search(search_term);
-      query = query
-        .filter(
+      query = if options.title_only.unwrap_or_default() {
+        query.filter(post::name.ilike(searcher))
+      } else {
+        query.filter(
           post::name
             .ilike(searcher.clone())
             .or(post::body.ilike(searcher)),
         )
-        .filter(not(post::removed.or(post::deleted)));
+      }
```

Comment:
Looks good. You could even bring the `post::name.ilike(searcher)` clause above this block, and put it above L390, since both of them use it.

Reply:
I tried implementing this but for some reason it broke the search logic.